pkg/server/server.go

```diff
@@ -227,7 +227,6 @@ func (s *Server) Check(ctx context.Context, req *openfgapb.CheckRequest) (*openf
 	if err != nil {
 		return nil, err
 	}
-	span.SetAttributes(attribute.KeyValue{Key: "authorization-model-id", Value: attribute.StringValue(modelID)})
```

Comment:
Please make this a `const` at the top of this package.

Reply:
Done